src/ProjectOrigin.WalletSystem.Server/appsettings.Development.json

```diff
@@ -10,5 +10,8 @@
     },
     "ServiceOptions": {
         "EndpointAddress": "http://localhost:80/"
+    },
+    "VerifySlicesWorkerOptions": {
+        "SleepTimeInSeconds" : 10
```

Comment:
No need to set this here in appsettings.development.json when it is the same value as in appsettings.json